quartz/plugins/transformers/links.ts

```diff
@@ -107,7 +107,7 @@ export const CrawlLinks: QuartzTransformerPlugin<Partial<Options> | undefined> =
 
                   // url.resolve is considered legacy
                   // WHATWG equivalent https://nodejs.dev/en/api/v18/url/#urlresolvefrom-to
-                  const url = new URL(dest, `https://base.com/${curSlug == "/" ? "" : curSlug}`)
+                  const url = new URL(dest, path.join("https://base.com/", curSlug))
```

Comment:
this function haha sorry https://github.com/jackyzha0/quartz/blob/a7325eadc1483c181928425a4c7379427f21995a/quartz/util/path.ts#L182

Reply:
Oh right duh, path.join would probably use backslashes on Windows. Fixed (and squashed)